test/toolchain/ScalaToolchainTest.scala

```diff
@@ -0,0 +1,86 @@
+package scala.test.toolchain
+
+import build.bazel.tests.integration.BazelBaseTestCase
+import org.specs2.mutable.{Before, SpecificationWithJUnit}
+import org.specs2.specification.{BeforeAll, Scope}
+import scala.collection.JavaConverters._
+
```

Comment:
I'm a little concerned that we are using specs for core testing here. Prior to this, all of our tests used scalatest and we only tested specs to exercise specifically specs. Can we have one consistent way of testing (which prior to this was scalatest)?